kotlin-js/src/jsMain/kotlin/js/typedarrays/Uint8Array.kt

```diff
@@ -18,5 +22,12 @@ open external class Uint8Array(
     constructor(elements: JsIterable<Byte>)
     constructor(elements: ReadonlyArray<Byte>)
 
+    inline fun asByteArray(): ByteArray =
+        Int8Array(buffer, byteOffset, length).unsafeCast<ByteArray>()
```

Comment:
`unsafeCast` used without reason

Reply:
@turansky I need the unsafe cast here, otherwise there is type mismatch.

Reply:
`Int8Array.asByteArray()`?

Reply:
Addressed.